examples/unreal_engine_crash.rs

```diff
@@ -0,0 +1,54 @@
+extern crate clap;
+extern crate failure;
+extern crate symbolic;
+
+use std::fs::File;
+use std::io::Read;
+use std::path::Path;
+
+use clap::{App, Arg, ArgMatches};
+use failure::Error;
+
+use symbolic::unreal::Unreal4Crash;
+
+fn execute(matches: &ArgMatches) -> Result<(), Error> {
+    let crash_file_path = matches.value_of("crash_file_path").unwrap();
+
+    let mut file = File::open(Path::new(crash_file_path))?;
```

Comment:
_nit:_ You can directly pass a string to `File::open`:

```suggestion
    let mut file = File::open(crash_file_path)?;
```